parquet-column/src/main/java/org/apache/parquet/schema/Float16.java

```diff
@@ -46,29 +46,10 @@
  * Ref: https://android.googlesource.com/platform/libcore/+/master/luni/src/main/java/libcore/util/FP16.java
  */
 public class Float16 {
```

Comment:
Please make any methods that are used only from the same package `package-private`.

Reply:
updated them to non-public, thanks!